### PR TITLE
Add test folder and start writing tests on menuNavigation

### LINKED ISSUE #23: Add test folder
It is important to get started with testing. Testing becomes one of the dev tools as it allows you to write robust cost and make sure that the code you write covers all the edge cases. Explore XCTest.

### PR DESCRIPTION
**Pull request by algebra2boy**
Went to File > Target > Unit Testing Bundle, created a new testing bundle for our app.

import XCTest
@testable import ShopHub

### CODE REVIEW

File: ShopHub/ShopHubTests/ShopHubTests.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
+//
```
Comment (TaoLyn838) [inline ShopHub/ShopHubTests/ShopHubTests.swift:1]: They are look great to me.

**Review by TaoLyn838 — APPROVED** [review]
